### DISCUSSION TITLE
What is difference  between Admin and Owner

### DISCUSSION THREAD

**Discussion by shota1998** [langgenius/dify#8583] [Help]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

We are using this application on-premise.   
The person who deployed Dify is currently the owner.   
Depending on the permissions, we would like to assign ownership to another appropriate person.

### 2. Additional context or comments

What I want to know this time is the difference between an administrator and an owner.

There is no direct way to transfer ownership that I can find.   
I am considering editing the database directly.

**Comment by nadirvishun** [comment]
One difference I've noticed is that `owner` can kick out members, but `admin` can't.

**Comment by yuzp-work** [comment]
Is there any method can handover the owner permission?

**Reply by y-omr** [reply]
Unfortunately, there is no way to transfer ownership rights.

https://docs.dify.ai/guides/management/team-members-management#frequently-asked-questions
